geonode/maps/urls.py

```diff
@@ -39,13 +39,17 @@
     map_embed = 'map_embed'
     # TODO qlr for geoserver
     map_download_qlr = 'map_download_qlr'
+    # TODO no implementation for geoserver
+    map_embed_widget = 'map_embed_widget'
```

Comment:
@boney-bun I'm thinking that for this we can just put `map_embed_widget` view into core geonode, because the frontend code is independent of either backend. I think it is ok to move map_embed_widget code from qgis_server_views to geonode.maps.views
CC @gubuntu